src/Theming/Fluent9DarkTheme.ts

```diff
@@ -0,0 +1,397 @@
+    colorNeutralShadowAmbient: 'rgba(0,0,0,0.12)',
+    colorNeutralShadowKey: 'rgba(0,0,0,0.14)',
+    colorNeutralShadowAmbientLighter: 'rgba(0,0,0,0.06)',
+    colorNeutralShadowKeyLighter: 'rgba(0,0,0,0.07)',
+    colorNeutralShadowAmbientDarker: 'rgba(0,0,0,0.20)',
+    colorNeutralShadowKeyDarker: 'rgba(0,0,0,0.24)',
+    colorBrandShadowAmbient: 'rgba(0,0,0,0.30)',
+    colorBrandShadowKey: 'rgba(0,0,0,0.25)',
+    colorPaletteRedBackground1: '#fdf6f6',
+    colorPaletteRedBackground2: '#f1bbbc',
+    colorPaletteRedBackground3: '#d13438',
+    colorPaletteRedForeground1: '#bc2f32',
+    colorPaletteRedForeground2: '#751d1f',
+    colorPaletteRedForeground3: '#d13438',
+    colorPaletteRedBorderActive: '#d13438',
+    colorPaletteRedBorder1: '#f1bbbc',
+    colorPaletteRedBorder2: '#d13438',
+    colorPaletteGreenBackground1: '#f1faf1',
+    colorPaletteGreenBackground2: '#9fd89f',
+    colorPaletteGreenBackground3: '#107c10',
+    colorPaletteGreenForeground1: '#0e700e',
+    colorPaletteGreenForeground2: '#094509',
+    colorPaletteGreenForeground3: '#107c10',
+    colorPaletteGreenBorderActive: '#107c10',
+    colorPaletteGreenBorder1: '#9fd89f',
+    colorPaletteGreenBorder2: '#107c10',
+    colorPaletteDarkOrangeBackground1: '#fdf6f3',
+    colorPaletteDarkOrangeBackground2: '#f4bfab',
+    colorPaletteDarkOrangeBackground3: '#da3b01',
+    colorPaletteDarkOrangeForeground1: '#c43501',
+    colorPaletteDarkOrangeForeground2: '#7a2101',
+    colorPaletteDarkOrangeForeground3: '#da3b01',
+    colorPaletteDarkOrangeBorderActive: '#da3b01',
+    colorPaletteDarkOrangeBorder1: '#f4bfab',
+    colorPaletteDarkOrangeBorder2: '#da3b01',
+    colorPaletteYellowBackground1: '#fffef5',
+    colorPaletteYellowBackground2: '#fef7b2',
+    colorPaletteYellowBackground3: '#fde300',
+    colorPaletteYellowForeground1: '#817400',
+    colorPaletteYellowForeground2: '#817400',
+    colorPaletteYellowForeground3: '#fde300',
+    colorPaletteYellowBorderActive: '#fde300',
+    colorPaletteYellowBorder1: '#fef7b2',
+    colorPaletteYellowBorder2: '#fde300',
+    colorPaletteBerryBackground1: '#fdf5fc',
+    colorPaletteBerryBackground2: '#edbbe7',
+    colorPaletteBerryBackground3: '#c239b3',
+    colorPaletteBerryForeground1: '#af33a1',
+    colorPaletteBerryForeground2: '#6d2064',
+    colorPaletteBerryForeground3: '#c239b3',
+    colorPaletteBerryBorderActive: '#c239b3',
+    colorPaletteBerryBorder1: '#edbbe7',
+    colorPaletteBerryBorder2: '#c239b3',
+    colorPaletteLightGreenBackground1: '#f2fbf2',
+    colorPaletteLightGreenBackground2: '#a7e3a5',
+    colorPaletteLightGreenBackground3: '#13a10e',
+    colorPaletteLightGreenForeground1: '#11910d',
+    colorPaletteLightGreenForeground2: '#0b5a08',
+    colorPaletteLightGreenForeground3: '#13a10e',
+    colorPaletteLightGreenBorderActive: '#13a10e',
+    colorPaletteLightGreenBorder1: '#a7e3a5',
+    colorPaletteLightGreenBorder2: '#13a10e',
+    colorPaletteMarigoldBackground1: '#fefbf4',
```

Comment:
Maybe we can have defaults in one file and then have our overrides in another variable or file so we know what we have changed from the default styles?